cogs/customreactions.py

```diff
@@ -0,0 +1,1111 @@
+            embed = discord.Embed(title="Loading...")
+            await message.edit(embed=embed)
+
+            custom_react = current_list[number - 1]
+            prompt = custom_react[1]
+            response = custom_react[2]
+            user_who_added = self.bot.get_user(custom_react[3])
+            delete = custom_react[4]
+            anywhere = custom_react[5]
+            dm = custom_react[6]
+            if delete == 1:
+                delete_str = "Deletes the message that calls the reaction"
+            else:
+                delete_str = "Does not delete the message that calls the reaction"
+            if anywhere == 1:
+                anywhere_str = "Activates the custom reaction if the prompt is anywhere in a message"
+            else:
+                anywhere_str = "Only activates the custom reaction if the prompt is the full message"
+            if dm == 1:
+                dm_str = "Reacts in the DMs of the user who calls the reaction instead of the channel"
+            else:
+                dm_str = "Reacts directly into the channel"
+
+            if proposals:
+                description = "1️⃣ Prompt: {}\n2️⃣ Response: {}\n3️⃣ {}\n4️⃣ {}\n5️⃣ {}" \
+                              "\n✅ Approve this proposal\n❌ Reject this proposal\nAdded by {}"\
+                              .format(prompt, response, delete_str, anywhere_str, dm_str, user_who_added)
+                title = "More information on a custom reaction proposal.\n" \
+                        "{}s may click on emojis to modify those values or approve/refuse this proposal\n" \
+                        "(Will return to the list of current reaction proposals in 40 seconds otherwise)"\
+                        .format(self.bot.config.moderator_role)
+            else:
+                description = "1️⃣ Prompt: {}\n2️⃣ Response: {}\n3️⃣ {}\n4️⃣ {}\n5️⃣ {}" \
+                              "\n🚮 Delete this custom reaction\nAdded by {}".format(prompt, response, delete_str,
+                                                                                    anywhere_str, dm_str,
+                                                                                    user_who_added)
+                title = "More information on a custom reaction.\n" \
+                        "{}s may click on emojis to modify those values or select an option\n" \
+                        "(Will return to the list of current reactions in 40 seconds otherwise)"\
+                        .format(self.bot.config.moderator_role)
+
+            embed = discord.Embed(title=title, description=description)
+            current_options.clear()
+            await message.clear_reactions()
+            if proposals:
+                current_options.extend(
+                    ["1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "✅", "❌", "⏹"])
+            else:
+                current_options.extend(
+                    ["1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "🚮", "⏹"])
```

Comment:
overlap with the above with 1-5, tuple since its not mutated

Reply:
Resolved in 7d16cb6